.github/pull_request_template.md

```diff
@@ -13,10 +13,11 @@
        Use `chore` to indicate that this PR makes some changes that users don't need to know.
     * `test: fix CI failure. Part of #1234`
         Use `part of` to indicate that a PR is working on an issue, but shouldn't close the issue when merged.
-
+-->
 - [ ] Do you want this pull request (PR) cherry-picked into the current release branch?
```

Comment:
Suggested revision (Since the second option does not require the approver to act, focus on asking the user specifically about adding the label.):

Do you want this pull request (PR) cherry-picked into the current release branch in the next batch update?

[Learn more about cherry-picking updates into the release branch](https://github.com/kubeflow/pipelines/blob/master/RELEASE.md#cherry-picking-pull-requests-to-release-branch).

Reply:
Makes sense! Updated accordingly